libs/wire-subsystems/test/unit/Wire/MockInterpreters/UserSubsystem.hs

```diff
@@ -7,9 +7,22 @@ import Wire.API.User
 import Wire.UserKeyStore
 import Wire.UserSubsystem
 
-userSubsystemTestInterpreter :: [UserAccount] -> InterpreterFor UserSubsystem r
+userSubsystemTestInterpreter :: [ExtendedUserAccount] -> InterpreterFor UserSubsystem r
 userSubsystemTestInterpreter initialUsers =
   interpret \case
-    GetLocalUserAccountByUserKey localUserKey -> case (tUnqualified localUserKey) of
-      EmailKey _ email -> pure $ find (\u -> userEmail u.accountUser == Just email) initialUsers
+    GetExtendedAccountsBy (tSplit -> (_dom, getBy)) ->
+      pure $
+        filter
+          ( \u ->
+              mailKeyFrom u
+                `elem` getBy.getByEmail
+          )
+          initialUsers
     _ -> error $ "userSubsystemTestInterpreter: implement on demand"
+
+mailKeyFrom :: ExtendedUserAccount -> EmailKey
+mailKeyFrom acc =
```

Comment:
users with invitation pending may need to be searched by their *unvalidated* email address in the extended account. not sure.